src/test-utils/mockDatePickers.tsx

```
@@ -0,0 +1,43 @@
+  value,
+  label = '',
+  disabled,
+  ...props
+}: InterfaceMockPickerProps) => {
+  const isTime = /\bTime\b/.test(label);
+  const format = isTime ? 'HH:mm:ss' : 'YYYY-MM-DD';
+  const dataTestId = isTime ? `time-picker-${label}` : `date-picker-${label}`;
+
+  return (
+    <div>
+      <label>{label}</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)